### PR TITLE
Jspsych response helpers

### PR DESCRIPTION
**Pull request by okaycj**
# Summary

For the experiment data coming out of jsPsych, we needed to update how the data was parsed.  This included adding a couple of conditionals and a shim until we can complete other issues.  

 - Added parsing logic to exit frame property function
 - For the time being, we'll not get consented responses.  This will be reverted when the consent view has been updated.
 - Handle jsPsych videos on save

# Notes 
This isn't the full solution to the researcher's data view.  Instead, this was a good place to check in before doing more work.

### CODE REVIEW

File: exp/tests/test_response_postsave.py [hunk 1]
```diff
@@ -208,3 +211,52 @@ def test_responses_per_study_type(self):
...
+        G(
+            Video,
+            response=self.response,
+            study=study,
+            s3_timestamp=timezone.now(),
+            frame_id=frame_id,
+        )
+
+    def test_exit_withdraw_videos(self):
+        settings.CELERY_TASK_ALWAYS_EAGER = True
```
Comment (becky-gilbert) [inline exp/tests/test_response_postsave.py:239]: Does this need to be set back to `False` at the end of the test or in a `tearDown` function?

**Review by becky-gilbert — APPROVED** [review]
LGTM! Just one question.

File: exp/tests/test_response_postsave.py (hunk 1 above)
Comment (okaycj) [reply exp/tests/test_response_postsave.py]: It didn't change the outcome of the tests, but I did it anyway.

Comment (becky-gilbert) [reply exp/tests/test_response_postsave.py]: Thanks!

**Pull request merged**